content/Google is a modal dialog - Martin Rubi.md

```diff
@@ -2,101 +2,101 @@
 
 ### The shame of being a modal dialog
 
-A modal dialog is a window which pops up on top of every other window freezing the
-underlaying user interface until the user dismisses it.
+Modal dialogs are windows that pops-up on top of every other window freezing the
+underlying user interface until the user dismisses them.
 
-Its use was quite frequent in the early days of desktop windows apps but with the
+Their use was quite frequent back in the early days of desktop windows apps but with the
 beginning of browser user intefaces they not only became obsolete but some articles
-were writen discouraging its use in favor of edit in place interfaces.
+discouraged their use in favor of "Edit-in-place" interfaces.
 
-I'm no UX expert but I am a user of graphics user intefaces on a daily basis and
+I'm no UX expert but I am a user of Graphical User Intefaces on a daily basis and
  as a user I will speak up in defense of modal dialogs when used properly.
 
 ### The importance of being modal
 
 What do modal dialogs do?
 
-When I first thought in modal dialogs I focused mainly on the space location
+When I first thought of modal dialogs I focused mainly on their space location
 in the user interface.
 
 The first desktop apps achieved 3D emulation by overlapping 2D rectangles called
-windows [1].
+windows[1].
 
 Browser interfaces avoid 3D emulation and usually stick to a strict 2D interface
-through laying out its elements instead of using overlapping.
+by laying out its elements instead of using overlapping.
 
 However this is just the implementation of what modal dialogs do and not its functional
 purpose and value.
 
-Modal dialogs interrupt the flow of the user attention to solve a particular task
+Modal dialogs interrupt the flow of the user's attention to solve a particular task
 providing a self contained, highly focused functional and visual context for the user.
 
-That is they provide for a very short period of time a user interface for the user
-to solve a specific task hidding every other complexity not related to that task.
+That is, they provide for a very short period of time a user interface for the user
+to solve a specific task hiding every other complexity not related to that task.
 
-Once the user ends solving that specific part of the app it returns to the previous
-flow just like she left it.
+Once the user ends solving that specific part of the app she returns to the previous
```

Comment:
Again, try to use non-gender-specific personal pronouns if the sentence does not require one. Switch `she` for `they` and `returns` for `return`

Reply:
I now see your answer to the resolved comment, as I said I would like to get the opinion of other revewers on this subject as we do not agree upon it.

Reply:
I agree with asking a second opinion and I will change the wording to that opinion.
However I can't help wondering if this sentence would have been an issue if I had used `he returns to previous` to refer to the gender of the user. I could bet it would
 had not.
I am all about gender equality but I just don't think anybody will achieve it by circumventing grammar constructs but by accepting that thinking a user as a`she` is as much as a good choice as thinking a user as a `he`